### PR TITLE
Addition of `fpr_dpq_clip` and `fpr_create_hydrograph`

### PR DESCRIPTION
**Pull request by newgraph-lschick**
- Addition of `fpr_dpq_clip` and `fpr_create_hydrograph`
- All tests passed. 
- Added description to param `dir_photos` in `fpr_photo_qa` to remove warning during build/check. 
- Updated NEWS.md.

- Updated both functions, checked and tested with devtools, both passed

### CODE REVIEW

**Comment by newgraph-lschick** [comment]
Updates have been made, ready for review

File: NAMESPACE [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 export("%>%")
 export(fpr_appendix_title)
-export(fpr_create_hydrograph)
+export(fpr_create_hydrograph_local)
```
Comment (NewGraphEnvironment) [inline NAMESPACE:5]: this name seems odd when cross referenced to the function...

File: man/fpr_create_hydrograph_local.Rd
[the remark this answers is not in the document]
Comment (newgraph-lschick) [reply man/fpr_create_hydrograph_local.Rd:1]: fixed this in latest commit

**Comment by newgraph-lschick** [comment]
Ready for review

**Review by NewGraphEnvironment — APPROVED** [review]
these are great but - as you know - lets use local functions in our repos for these for a while until they are more stable.

**Pull request merged**